src/Microsoft.CodeAnalysis.Razor.Workspaces/ErrorReporter.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.CodeAnalysis.Razor
+{
+    internal abstract class ErrorReporter : IWorkspaceService
+    {
+        public abstract void ReportError(Exception exception);
+
+        public abstract void ReportError(Exception exception, Project project);
```

Comment:
Yeah, i'm specifically formatting the errors to say that the problem is with "Project X"

This can change in the future if we need it, it's not public